### DISCUSSION TITLE
How to find The Appcellerator Studio intall File for macOS

### DISCUSSION THREAD

**Discussion by cloncle35** [appcelerator/titanium_mobile#12958] [Q&A] [answered]
Hi everybody,

Since the beginning of the week I tried to find the install file for Appcelerator Studio (for macOS version 11.4) and impossible to find it. I don't understand how to do it ? I have a old version of Appcelerator on a old macOS and I want to run my project on a most recent environnent (on a new Mac) and first i have to do is to install appcelerator Studio (I have already install nodeJS). Thanks a lot for your help. Cyrille

**Comment by m1ga** [comment, marked as answer]
Please use either Atom or VSCode with the official packages. Appc Studio has been discontinued and the other options e.g. use less memory and are faster. Plus you can use other extensions too
